packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -248,6 +248,19 @@ export class TableQuadrantStack extends AbstractComponent<ITableQuadrantStackPro
         this.syncQuadrantViews();
     }
 
+    /**
+     * Synchronizes quadrant sizes and scroll offsets based on the current
+     * column, row, and header sizes. Useful for correcting quadrant sizes after
+     * explicitly resizing columns and rows, for instance.
+     *
+     * Invoking this method imperatively is cheaper than providing columnWidths
+     * or rowHeights array props to TableQuadrantStack and forcing it to run
+     * expensive array diffs upon every update.
```

Comment:
are you certain it's less expensive to make dom measurements in `syncQuadrantViews` than to diff some array values? won't it force a layout? what if the layout is slow?

Reply:
We invoke this method only when `didColumnOrRowSizesChangeInLatestUpdate=true`, so we're doing reflows only when we have to.

On a related/unrelated note, I'm not positive I handled controlled-mode changes correctly: i.e., what happens when you pass in a new `rowHeights` or `columnWidths` array to the `<Table>` component? Need to make sure I update the quadrant sizes appropriately in that case.

Reply:
Turns out we don't honor controlled `columnWidths` or `rowHeights` at all after the first render... Therefore not a blocker for this PR, but we should fix it.